ui-admin/src/portal/PortalProvider.tsx

```diff
@@ -7,13 +7,15 @@ import { NavBreadcrumb } from 'navbar/AdminNavbar'
 
 export type PortalContextT = {
   updatePortal: (portal: Portal) => void
+  reloadPortal: (shortcode: string, onReload: () => void) => void,
```

Comment:
This should return a promise rather than take a callback for easier use
```suggestion
  reloadPortal: (shortcode: string) => Promise<Portal>,
```

Reply:
Got rid of callback in 96d4f97f